1-js/02-first-steps/14-switch/1-rewrite-switch-if-else/task.md

```diff
@@ -1,26 +1,25 @@
-importance: 5
+اهمیت: 5
```

Comment:
As you can see in this [link](https://github.com/javascript-tutorial/fa.javascript.info#metadata), You should not translate metadata like `importance`.